src/app/components/pages/SubjectOverviewPage.tsx

```diff
@@ -112,15 +149,28 @@ export const SubjectOverviewPage = withRouter((props: RouteComponentProps) => {
 
             <p className="mt-3">
                 All Isaac Science questions are classed as either &quot;Practice&quot; or &quot;Challenge&quot; – indicated by the symbols below. 
+            </p>
+            
+            <div className="d-flex flex-row w-100 justify-content-center">
+                <div className="d-flex flex-column me-3 align-items-center">
+                    <DifficultyIcon difficultyCategory="P"/>
+                    <span>Practice</span>
+                </div>
+                <div className="d-flex flex-column align-items-center">
+                    <DifficultyIcon difficultyCategory="C"/>
+                    <span>Challenge</span>
+                </div>
+            </div>
 
+            <p className="mt-3">
                 In Isaac {humanSubject},
                 <ul>
                     <li>Practice questions are those that require one concept or equation to solve.</li>
                     <li>Challenge questions are those that require one or more concepts, or require more creativity to solve the problem, helping to develop important problem solving skills. </li>
                 </ul>
             </p>
 
-            {/* <ExampleQuestions/> */}
+            <ExampleQuestions subject={pageContext.subject} />
```

Comment:
```suggestion
            <div className="d-flex justify-content-center">
                <div className="w-100 w-md-50">
                    <ExampleQuestions subject={pageContext.subject} />
                </div>
            </div>
```
The mockups have this roughly half-width and centered, which imo does look better for bigger screens (especially with the difficulty icons above centered anyway). Happy to merge as-is if you disagree, though.

Reply:
Hrm, I partly agree. I think we can make this code itself cleaner by passing the new `className` into the `ListView` directly rather than two layers of divs - however more notably I think that frequently switching width between 100->50->100->50 is quite jarring, but we can ease that and create a better flow than either other option by using `w-75`.

I assume this still matches your wishes nicely, but if it _doesn't_, I'd prefer `w-100` to `w-50`.